src/main/scala/com/databricks/spark/avro/package.scala

```diff
@@ -15,7 +15,12 @@
  */
 package com.databricks.spark
 
-import org.apache.spark.sql.{SQLContext, DataFrameReader, DataFrameWriter, DataFrame}
+import org.apache.avro.generic.GenericRecord
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql._
+import scala.collection.JavaConversions._
```

Comment:
Nit: import ordering. Please group the Scala imports in their own section prior to the other imports.